### PR TITLE
French translation

### PR DESCRIPTION
**Pull request by Nilvus**
First commit of french translation.
Next commit will follow for colorbalance rgb module.

@TurboGit: if you want to check first commit before.

### CODE REVIEW

**Comment by Nilvus** [comment]
@aurelienpierre: could you especially check 2nd commit of this PR. I'm quite sure some translations are not really good. Thanks.

**Comment by Nilvus** [comment]
@aurelienpierre: my bad, I didn't split my work correctly. Check also 1st commit starting line 10817.

File: po/fr.po [hunk 1]
```diff
@@ -10084,99 +10190,107 @@ msgstr ""
...
 
-#: ../src/iop/channelmixerrgb.c:3510
+#: ../src/iop/channelmixerrgb.c:3568
 msgid "sky and water colors"
 msgstr "couleurs du ciel et de l'eau"
 
-#: ../src/iop/channelmixerrgb.c:3512
+#: ../src/iop/channelmixerrgb.c:3569
+msgid "average delta E"
+msgstr "average delta E moyen"
```
Comment (TurboGit) [inline po/fr.po:10287]: supprimer average

File: po/fr.po [hunk 2]
```diff
@@ -6057,6 +6126,41 @@ msgid_plural "%s have been imported"
...
+#, c-format
+msgid "size: %3.2f%%"
+msgstr "taille : %3.2f%%"
+
+#: ../src/develop/masks/circle.c:94 ../src/develop/masks/circle.c:144
+#: ../src/develop/masks/ellipse.c:420 ../src/develop/masks/ellipse.c:505
+#: ../src/develop/masks/path.c:938 ../src/develop/masks/path.c:945
+#, c-format
+msgid "feather size: %3.2f%%"
+msgstr "taille d'adoucissement : %3.2f%%"
```
Comment (TurboGit) [inline po/fr.po:6147]: I would use just "adoucissement : %3.2f%%"

This is a transient toast message, shorter will be easier to read and it seems clear that we are talking about size here.

File: po/fr.po [hunk 3]
```diff
@@ -10699,6 +10814,149 @@ msgstr "ajuste l'histogramme et centre la luminosité moyenne"
...
+
+#: ../src/iop/colorbalancergb.c:773
+#, fuzzy
+msgid "luminance gain in highlights"
+msgstr "ajouter du bruit dans les hautes lumières"
+
+#: ../src/iop/colorbalancergb.c:780
+#, fuzzy
+msgid "hue of the color gain in highlights"
+msgstr "change le contraste local des hautes lumières"
```
Comment (TurboGit) [inline po/fr.po]: change la teinte des ...

File: po/fr.po [hunk 4]
```diff
@@ -10699,6 +10814,149 @@ msgstr "ajuste l'histogramme et centre la luminosité moyenne"
...
+
+#: ../src/iop/colorbalancergb.c:788
+#, fuzzy
+msgid "chroma of the color gain in highlights"
+msgstr "change le contraste local des hautes lumières"
```
Comment (TurboGit) [inline po/fr.po]: change la saturation des ...

File: po/fr.po [hunk 5]
```diff
@@ -13775,6 +14013,14 @@ msgstr "force du flou pour les couleurs"
 msgid "overexposed"
 msgstr "sous- et sur-exposition"
 
+#: ../src/iop/overexposed.c:128 ../src/iop/overexposed.c:416
+msgid "module overexposed failed in buffer allocation"
+msgstr "le module surexposé a échoué dans l'allocation de la mémoire tampon"
+
+#: ../src/iop/overexposed.c:154 ../src/iop/overexposed.c:426
+msgid "module overexposed failed in color conversion"
+msgstr "le module surexposé a échoué dans la conversion des couleurs"
```
Comment (TurboGit) [inline po/fr.po:14018]: surexposé -> sur-exposition (ou de surexposition) 

(le nom du module ici)

File: po/fr.po [hunk 6]
```diff
@@ -13775,6 +14013,14 @@ msgstr "force du flou pour les couleurs"
 msgid "overexposed"
 msgstr "sous- et sur-exposition"
 
+#: ../src/iop/overexposed.c:128 ../src/iop/overexposed.c:416
+msgid "module overexposed failed in buffer allocation"
+msgstr "le module surexposé a échoué dans l'allocation de la mémoire tampon"
```
Comment (TurboGit) [inline po/fr.po:14014]: idem ci-dessous

**Review by TurboGit — CHANGES_REQUESTED** [review]
Some comments. When fixed I'll merge and will handle the import dialog. This requires a changes in XSL, dtd and config and need more testing. I'll do that on top of that to not break this work with conflicts.

File: po/fr.po (hunk 1 above)
Comment (Nilvus) [reply po/fr.po:10287]: oups, un oubli !

File: po/fr.po (hunk 2 above)
Comment (Nilvus) [reply po/fr.po:6147]: ok

File: po/fr.po (hunk 3 above)
Comment (Nilvus) [reply po/fr.po]: il me semblait l'avoir corrigé celui-là... un mauvais copier-coller

File: po/fr.po (hunk 5 above)
Comment (Nilvus) [reply po/fr.po:14018]: pas compris ce que tu entends par (le nom du module ici)

File: po/fr.po (hunk 3 above)
Comment (Nilvus) [reply po/fr.po]: je confirme il est corrigé (dans le 2ème commit) et indiqué "outdated"

File: po/fr.po (hunk 4 above)
Comment (Nilvus) [reply po/fr.po]: idem, corrigé dans 2ème commit

**Comment by Nilvus** [comment]
> Some comments. When fixed I'll merge and will handle the import dialog. This requires a changes in XSL, dtd and config and need more testing. I'll do that on top of that to not break this work with conflicts.

Comments fixed. Just the last 2 ones I'm not sure to understand. See if my last commit answer it correctly or not.

2 comments you made were already fixed in 2nd commit. So you could review. I don't see @aurelienpierre review, so you merge it as it is or wait, as you want.

File: po/fr.po (hunk 5 above)
Comment (TurboGit) [reply po/fr.po]: Le nom du module n'est pas "sur-exposé" mais "sur-exposition".

File: po/fr.po [hunk 7]
```diff
@@ -14011,11 +14011,12 @@ msgstr "sous- et sur-exposition"
...
 #: ../src/iop/overexposed.c:128 ../src/iop/overexposed.c:416
 msgid "module overexposed failed in buffer allocation"
-msgstr "le module surexposé a échoué dans l'allocation de la mémoire tampon"
+msgstr ""
+"la surexposition du module a échoué dans l'allocation de la mémoire tampon"
 
 #: ../src/iop/overexposed.c:154 ../src/iop/overexposed.c:426
 msgid "module overexposed failed in color conversion"
-msgstr "le module surexposé a échoué dans la conversion des couleurs"
+msgstr "la surexposition du module a échoué dans la conversion des couleurs"
```
Comment (TurboGit) [inline po/fr.po:14019]: le module sur-exposition a échoué...

File: po/fr.po [hunk 8]
```diff
@@ -14011,11 +14011,12 @@ msgstr "sous- et sur-exposition"
 
 #: ../src/iop/overexposed.c:128 ../src/iop/overexposed.c:416
 msgid "module overexposed failed in buffer allocation"
-msgstr "le module surexposé a échoué dans l'allocation de la mémoire tampon"
+msgstr ""
+"la surexposition du module a échoué dans l'allocation de la mémoire tampon"
```
Comment (TurboGit) [inline po/fr.po:14015]: le module sur-exposition a échoué...

File: po/fr.po (hunk 7 above)
Comment (Nilvus) [reply po/fr.po:14019]: Ok, mais il est où ce module? Je ne vois même pas à quelle partie de darktable ça correspond ces 2 lignes. Si tu peux m'indiquer, je serais curieux de comprendre. Dans tous les cas, je corrige.

Comment (TurboGit) [reply po/fr.po:14019]: Il est visible en tant qu'icone unique dans la barre en bas dans la chambre noire.

Comment (Nilvus) [reply po/fr.po]: Ah ok, c'est ça. Merci, j'avais pas fait le lien.

**Comment by Nilvus** [comment]
Corrections faites @TurboGit. Prêt pour revu et intégration si ok.

**Comment by TurboGit** [comment]
Thanks, so I'll merge now and will finalize the work for the missing translated strings. Thanks a lot !

**Pull request merged**